apps/settings/js/modules/async_storage.js

```diff
@@ -16,6 +16,11 @@ define(function(require) {
       return new Promise(function(resolve, reject) {
         AsyncStorage.setItem(key, value, resolve);
       });
+    },
+    removeItem: function as_removeItem(key) {
```

Comment:
Note: I think I no need to do the change because I `require('shared/async_storage');`. Will remove it later.